xbmc/music/MusicUtils.cpp

```diff
@@ -638,11 +639,70 @@ void ShowToastNotification(const CFileItem& item, int titleId)
   CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Info, g_localizeStrings.Get(titleId),
                                         message);
 }
+
+std::string GetMusicDbItemPath(const CFileItem& item)
+{
+  std::string path = item.GetPath();
```

Comment:
shoudn't it be DynPath here? Afaik most of the issues with "play next x automatically" are due to the use of path instead of the resolved one

Reply:
AFAIK, `GetPath` is correct here, because here we are interested in the identity of the item, not what actually we be played.